### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by fed1k**
* Remove current_user method in ApplicationController, devise will provide us with one.
* [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
  * Can register a new user.
  * User logs in with a combination of email and password.
  * Hashed password should be stored in the database.
  * Ask for confirmation of email.
  * Can reset password.
* [Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and your styling.

### CODE REVIEW

File: app/models/member.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Member < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+end
```
Comment (OLIPLICHE) [inline app/models/member.rb:6]: - [ ] Inside Model user.rb:
-  Please you need to enable `:registerable`, `:database_authenticable`, `:recoverable` the code will look like this but not exactly the same : 
  `devise :database_authenticatable, :registerable,
	      :recoverable, :rememberable, :validatable, :confirmable`
- 
Capture
- From the request above: The application allows for registering a new user, hashed password is stored in the database,  user can reset the password in the application

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Linter is green (the last commit have a green check mark (✅)) 👏
README file is descriptive 👏
Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
- [x] INSTALL AND SET UP DEVISE: Please make sure Devise is well installed and set up:
-  I noticed that you missed a line of code similar to this ` config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }` inside your `config/environments/development.rb`. Kindly review this if possible to make your App work fine. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/member.rb (hunk 1 above)
Comment (fed1k) [reply app/models/member.rb:6]: Hello @OLIPLICHE Thanks for the hints. You said like this but not exactly same. In your hint you mentioned everything that devise has and said not exactly same. So should I add some more extra things besides what you showed ?

Comment (OLIPLICHE) [reply app/models/member.rb:6]: Yes, Please you can add. I used "not exactly the same" to give the opportunity to freely write the code as you want. just make sure it works based on the requirements of the project.

**Review by AshakaE — APPROVED** [review]
Hi @fed1k , 

Thanks for making the changes 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**